### ISSUE TITLE
Improvements to /learn 

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [freeCodeCamp/freeCodeCamp#38781]
Here are the discussed improvements: 
- [ ] add a button to the button stack to take user to the next challenge if the user has already completed some challenges.
- [ ] remove "If you are new to coding, we recommend you start at the beginning." message if the user has completed a certain number of challenges.

**Comment by Upinder123** [comment]
i can help with this issue

**Comment by ahmadabdolsaheb** [comment]
sure, @Upinder123. I am getting these changes approved. They should be up for grabs in a day.

**Comment by thecodingaviator** [comment]
@ahmadabdolsaheb Even though I'm already logged in, when I go to fcc.org and click on the CTA, I'm redirected to the login page. We need a redirect to `\learn` if user is logged-in and not already present

**Comment by Twaha-Rahman** [comment]
> * [ ]  remove clutter from learn (most likely to shared elements between landing and learn should be removed) to make it learning focused

What do you mean by clutter? Can you please give me an example?

**Comment by Twaha-Rahman** [comment]
@ahmadabdolsaheb I've figured out how to check if a user has completed a challenge or not but how are we going to redirect a user to the next page?

**Comment by ahmadabdolsaheb** [comment]
> > * [ ]   remove clutter from learn (most likely to shared elements between landing and learn should be removed) to make it learning focused
> 
> What do you mean by clutter? Can you please give me an example?

You are right. it was ambiguous,  so I removed it for now until we have a more concrete direction.


>I've figured out how to check if a user has completed a challenge or not but how are we going to redirect a user to the next page?

No redirection is needed here. Just a button with a link to the right challenge.
If you need this functionality for the front page issue, I removed that feature request for now.

**Comment by ahmadabdolsaheb** [comment]
> @ahmadabdolsaheb Even though I'm already logged in, when I go to fcc.org and click on the CTA, I'm redirected to the login page. We need a redirect to `\learn` if user is logged-in and not already present

Hmm, I am not sure if I am understanding this correctly, but login.js should navigate to learn if you are signed in.

`const createOnClick = (navigate, isSignedIn) => e => {
  e.preventDefault();
  gtagReportConversion();
  if (isSignedIn) {
    return gatsbyNavigate('/learn');
  }
  return navigate(`${apiLocation}/signin`);
};`

**Comment by thecodingaviator** [comment]
@ahmadabdolsaheb apologies. I missed this.

When I'm on:
![image](https://user-images.githubusercontent.com/34807532/85573295-63bb9500-b653-11ea-8711-7fa4e45cc8e5.png)
 it does not redirect me to https://www.freecodecamp.org/learn even if I'm signed in.

**Comment by ahmadabdolsaheb** [comment]
That is by design, @thecodingaviator. Quincy wanted the landing page to be independent.
